fhir-path/src/main/java/org/linuxforhealth/fhir/path/FHIRPathQuantityNode.java

```diff
@@ -72,6 +115,38 @@ public String getQuantityCode() {
     public String getQuantityUnit() {
         return quantityUnit;
     }
+    
+    /**
+     * @implSpec This method is introduced to return the quantity sub type if a valid quantity sub type is
+     *      wrapped by this FHIRPathQuantityNode. 
+     *      If the type of FHIRPathQuantityNode is quantity, then return quantity type.
```

Comment:
```suggestion
     *      If the FHIRPathQuantityNode has no subtype, then return quantity type.
```

Reply:
This has been fixed. The java doc has been updated as suggested.